packages/batch_jaro_winkler/batch_jaro_winkler.0.1.1/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "2.0"
+maintainer: "Dominik Bousquet <bousquet.dominik@gmail.com>"
+authors: [ "Dominik Bousquet" ]
+license: "MIT"
+homepage: "https://github.com/dbousque/batch_jaro_winkler"
+dev-repo: "git://github.com/dbousque/batch_jaro_winkler.git"
+bug-reports: "https://github.com/dbousque/batch_jaro_winkler/issues"
+synopsis: "Fast batch jaro winkler distance implementation in C99"
+description: """
+This project gets its performance from the pre-calculation of an optimized
+model in advance of the actual runtime calculations. Supports any encoding.
+"""
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
```

Comment:
```suggestion
  ["dune" "build" "-p" name "-j" jobs "--root=./ocaml/lib"]
```

Reply:
I think with your current structure you need to tell dune where to look for the sources